docs/tutorial/part03.rst

```diff
@@ -2,5 +2,245 @@
+        countries      = StringProperty()
+        node_id        = StringProperty()
+        officers       = RelationshipFrom('.officer.Officer', 'REGISTERED_ADDRESS')
+        intermediaries = RelationshipFrom('.intermediary.Intermediary', 'REGISTERED_ADDRESS')   
+
+
+intermediary.py::
+
+    from neomodel import (
+        StringProperty,
+        StructuredNode,
+        RelationshipTo
+    )
+
+    class Intermediary(StructuredNode, NodeUtils):
+        sourceID      = StringProperty()
+        valid_until   = StringProperty()
+        name          = StringProperty()
+        country_codes = StringProperty()
+        countries     = StringProperty()
+        node_id       = StringProperty()
+        status        = StringProperty()
+        entities      = RelationshipTo('.entity.Entity', 'INTERMEDIARY_OF')
+        addresses     = RelationshipTo('.address.Address', 'REGISTERED_ADDRESS')
+
+
+
+officer.py::
+
+    from neomodel import (
+        StringProperty,
+        StructuredNode,
+        RelationshipTo,
+    )
+
+    class Officer(StructuredNode):
+        sourceID      = StringProperty()
+        name          = StringProperty()
+        country_codes = StringProperty()
+        valid_until   = StringProperty()
+        countries     = StringProperty()
+        node_id       = StringProperty()
+        addresses     = RelationshipTo('.address.Address', 'REGISTERED_ADDRESS')
+        entities      = RelationshipTo('.entity.Entity', 'OFFICER_OF')
+
+other.py::
+
+    from neomodel import (
+        StringProperty,
+        StructuredNode,
+        RelationshipTo,
+    )
+
+    class Other(StructuredNode):
+        sourceID    = StringProperty()
+        name        = StringProperty()
+        valid_until = StringProperty()
+        node_id     = StringProperty()
+        countries   = StringProperty()
+        addresses   = RelationshipTo('.address.Address', 'REGISTERED_ADDRESS')
+
+Modify the __init__.py file:
+==================================
+Now we need to add a little bit of code to the __init__.py. If you noticed in the previous code, we refer to other classes but we did no import them. e.g : ``.officer.Officer``. This means that it should not work because the program doesn't know what that is; Therefore, we are going to add all the imports to the init file so that each model has all the imports ::
+
+    from .entity import Entity
+    from .address import Address
+    from .intermediary import Intermediary
+    from .officer import Officer
+    from .other import Other
+
+
+    MODEL_ENTITIES = {
+        'Entity': Entity,
+        'Address': Address,
+        'Intermediary': Intermediary,
+        'Officer': Officer,
+        'Other': Other
+    } 
+
+
+Create constraints or indexes: 
+==================================
+
+Creating constraints and labels have to be done after you add/change the node definitions.
+The command that you will need to use is: 
+
+    ``python manage.py install_labels``

```

Comment:
In this case we are only reading the Paradise Paper DB, so does it really have any effect? 

Maybe we should point that we are just mapping an existing Graph Database, but we can leave it here so they know that   if `install_labels` is executed in an clean DB, Neomodel will create the node labels, properties and relationships; as well as  any constraints or indexes.